index.js

```diff
@@ -3,16 +3,18 @@ const execa = require('execa');
 const stripAnsi = require('strip-ansi');
 const defaultShell = require('default-shell');
 
-const args = ['-ilc', 'env; exit'];
+const args = ['-ilc', 'echo -n "_SHELL_ENV_DELIMITER_"; env; echo -n "_SHELL_ENV_DELIMITER_"; exit'];
 
 function parseEnv(env) {
+	env = env.split('_SHELL_ENV_DELIMITER_')[1];
 	const ret = {};
 
 	for (const line of stripAnsi(env).split('\n')) {
-		const parts = line.split('=');
-		ret[parts.shift()] = parts.join('=');
+		if (line !== '') {
+			const parts = line.split('=');
+			ret[parts.shift()] = parts.join('=');
+		}
```

Comment:
Suggestion:

````js
 	for (const line of stripAnsi(env).split('\n').filter(line => !!line)) {
````

Reply:
Hi, I just test this code, but after` npm test `, it throws:
```
index.js:12:63
  ✖  12:63  use Boolean(line) instead.  no-implicit-coercion
```
so how about :
```
for (const line of stripAnsi(env).split('\n').filter(line => Bealoon(line))) {
```

Reply:
Ah, yes. `Boolean(line)` instead of `!!line`.